samples/matter/common/src/bridge/bridge_manager.h

```diff
@@ -6,41 +6,62 @@
 
 #pragma once
 
+#include "bridge_util.h"
 #include "bridged_device.h"
 #include "bridged_device_data_provider.h"
-#include <map>
 
 class BridgeManager {
 public:
 	void Init();
-	CHIP_ERROR AddBridgedDevice(BridgedDevice::DeviceType bridgedDeviceType, const char *nodeLabel);
+	CHIP_ERROR AddBridgedDevices(BridgedDevice *aDevice, BridgedDeviceDataProvider *aDataProvider);
```

Comment:
OK, it still represents a single physical device, so that's why it confused me, but I guess the current name is fine, but maybe in the future you could add some doxygen comment about that :).